Hyper/Controllers/SkyboxController.cs

```diff
@@ -0,0 +1,229 @@
+﻿using System.Diagnostics;
+using Common;
+using Hyper.Shaders.DataTypes;
+using Hyper.Shaders.ModelShader;
+using Hyper.Shaders.ObjectShader;
+using Hyper.Shaders.SkyboxShader;
+using OpenTK.Mathematics;
+
+namespace Hyper.Controllers;
+internal class SkyboxController : IController
+{
+    private readonly Scene _scene;
+
+    private readonly AbstractSkyboxShader _skyboxShader;
+
+    private readonly StandardModelShader _modelShader;
+
+    private readonly StandardObjectShader _objectShader;
+
+    private readonly Skybox.Skybox _skybox;
+
+    private readonly Stopwatch _stopwatch = new();
+
+    private readonly float _dayLengthSeconds = 60 * 10; // length of the day in seconds
```

Comment:
If you don't think we need the comment just change tha variable name and remove the comment. same for the one below

Reply:
removed the stupid comments, they were basically variable names reiterated